newrelic/resource_newrelic_nrql_alert_condition_test.go

```diff
@@ -25,14 +25,14 @@ func TestAccNewRelicNrqlAlertCondition_Basic(t *testing.T) {
 		Steps: []resource.TestStep{
 			// Test: Create
 			{
-				Config: testAccNewRelicNrqlAlertConditionConfigBasic(rName, "20", "120", "sTaTiC", "0", "", "60"),
+				Config: testAccNewRelicNrqlAlertConditionConfigBasic(rName, "20", "120", "sTaTiC", "0", "", "60", "30"),
```

Comment:
:memo: There are a lot of positional arguments used in this test. If you would like me to convert these positional arguments to a single struct, I can do that for you.